build/ci/run-e2e-tests.yml

```diff
@@ -40,8 +40,11 @@ jobs:
         $cxt = New-AzStorageContext -StorageAccountName '$(azureStorageAccountName)' -StorageAccountKey $primaryKey
         $token = New-AzStorageAccountSASToken -Service Blob -ResourceType Container,Object -Permission 'rwdl' -Start $start -ExpiryTime $end -Context $cxt
 
+        $blobContainerName = (New-Guid).ToString('D')
```

Comment:
Name guidelines: https://learn.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names